test/test_config.py

```diff
@@ -0,0 +1,31 @@
+"""Tests related to ansiblelint.config module."""
+import os
+import subprocess
+import sys
+from pathlib import Path
+
+
+def test_call_from_outside_venv() -> None:
```

Comment:
I think the copy-paste could be reduced with `@pytest.mark.parametrize` since it's essentially one test. This would also help make the test variants more accurate — the first one relies on unstable side-effects, for example.

Reply:
Done, but it was not really necessary.